Index: content/browser/web_contents/web_contents_impl.cc
===================================================================
--- content/browser/web_contents/web_contents_impl.cc	(revision 235101)
+++ content/browser/web_contents/web_contents_impl.cc	(working copy)
@@ -3590,7 +3590,9 @@
   if (rvh != GetRenderViewHost())
     return;
 
+#if 0
   RenderViewHostImpl* rvhi = static_cast<RenderViewHostImpl*>(rvh);
+#endif
 
   // Ignore renderer unresponsive event if debugger is attached to the tab
   // since the event may be a result of the renderer sitting on a breakpoint.
@@ -3598,6 +3600,7 @@
   if (DevToolsAgentHost::IsDebuggerAttached(this))
     return;
 
+#if 0
   if (is_during_beforeunload || is_during_unload) {
     // Hang occurred while firing the beforeunload/unload handler.
     // Pretend the handler fired so tab closing continues as if it had.
@@ -3620,6 +3623,7 @@
       Close(rvh);
     return;
   }
+#endif
 
   if (!GetRenderViewHostImpl() || !GetRenderViewHostImpl()->IsRenderViewLive())
     return;
