Index: content/browser/web_contents/render_view_host_manager.cc
===================================================================
--- content/browser/web_contents/render_view_host_manager.cc	(revision 235101)
+++ content/browser/web_contents/render_view_host_manager.cc	(working copy)
@@ -627,6 +627,14 @@
     return SiteInstance::CreateForURL(browser_context, dest_url);
   }
 
+  // The user wants a restart of renderer process.
+  if (GetContentClient()->browser()->ShouldSwapProcessesForNavigation(
+          render_view_host_->GetSiteInstance(),
+          current_url,
+          dest_url)) {
+    return SiteInstance::CreateForURL(browser_context, dest_url);
+  }
+
   // Use the current SiteInstance for same site navigations, as long as the
   // process type is correct.  (The URL may have been installed as an app since
   // the last time we visited it.)
