Index: base/process/memory_mac.mm
===================================================================
--- base/process/memory_mac.mm	(revision 273293)
+++ base/process/memory_mac.mm	(working copy)
@@ -719,8 +719,10 @@
         << "Failed to get kCFAllocatorMallocZone allocation function.";
     context->allocate = oom_killer_cfallocator_malloc_zone;
   } else {
+#if 0
     NSLog(@"Internals of CFAllocator not known; out-of-memory failures via "
         "CFAllocator will not result in termination. http://crbug.com/45650");
+#endif
   }
 #endif
 
