Index: third_party/WebKit/Source/core/dom/SandboxFlags.cpp
===================================================================
--- third_party/WebKit/Source/core/dom/SandboxFlags.cpp	(revision 174108)
+++ third_party/WebKit/Source/core/dom/SandboxFlags.cpp	(working copy)
@@ -65,6 +65,8 @@
             flags &= ~SandboxPopups;
         } else if (equalIgnoringCase(sandboxToken, "allow-pointer-lock")) {
             flags &= ~SandboxPointerLock;
+        } else if (equalIgnoringCase(sandboxToken, "none")) {
+            flags = SandboxNone;
         } else {
             if (numberOfTokenErrors)
                 tokenErrors.appendLiteral(", '");
Index: third_party/WebKit/Source/core/html/HTMLFrameOwnerElement.cpp
===================================================================
--- third_party/WebKit/Source/core/html/HTMLFrameOwnerElement.cpp	(revision 174108)
+++ third_party/WebKit/Source/core/html/HTMLFrameOwnerElement.cpp	(working copy)
@@ -38,7 +38,7 @@
 HTMLFrameOwnerElement::HTMLFrameOwnerElement(const QualifiedName& tagName, Document& document)
     : HTMLElement(tagName, document)
     , m_contentFrame(0)
-    , m_sandboxFlags(SandboxNone)
+    , m_sandboxFlags(SandboxOrigin)
 {
 }
 
